src/helpers/OTHelper.js

```diff
@@ -77,8 +79,8 @@ const logRequest = (body) => {
     });
 };
 
-const logOT = (apiKey, sessionId, action, connectionId) => {
-  logRequest(getLog(apiKey, sessionId, action, connectionId));
+const logOT = (apiKey, sessionId, action, connectionId, proxyUrl) => {
```

Comment:
This implementation might break logging because connectionId is an optional value. Please see: https://github.com/opentok/opentok-react-native/blob/master/src/OTSession.js#L34

Reply:
Could you please validate?